mcs/class/System/System.Net.NetworkInformation/UnixIPGlobalProperties.cs

```diff
@@ -57,8 +57,19 @@ static int getdomainname ([MarshalAs (UnmanagedType.LPArray, SizeParamIndex = 1)
 		public override string DomainName {
 			get {
 				byte [] bytes = new byte [256];
-				if (getdomainname (bytes, 256) != 0)
-					throw new NetworkInformationException ();
+#if UNITY
```

Comment:
Hmm, I feel like we started with this in our previous branch and then made a better fix by check platform rather than exceptions?

Reply:
Yes, I was thinking of this subsequent PR https://github.com/Unity-Technologies/mono/pull/1199

Reply:
It looks like the difference is that the code in https://github.com/Unity-Technologies/mono/pull/1199 does the same thing without an exception, is that correct? I don't see a behavior difference. (I'm just trying to understand that PR).